.ibm/pipelines/auth/secrets-rhdh-secrets.yaml

```diff
@@ -4,6 +4,8 @@ metadata:
   name: rhdh-secrets
 data:
   # the followings are place holders with dummy values that will be overwritten by CI
+  GITHUB_APP_CLIENT_ID_FLEX: dGVtcA==
```

Comment:
what is GITHUB_APP_CLIENT_ID_FLEX ?

Reply:
Since there is now more than one app that we use to house the callback url, I created this to be the CLIENT_ID used for authentication, so we can keep the CLIENT_ID used for GH integration the same across all projects.

Reply:
I think I overthought the GH integration tests. I realize I can use APP_3 secrets for the GH org integration as well since APP_3 and APP_2 and APP_1 share the same org.